docs/md/checkpoint_learn_example_2_nonintrusive.md

```diff
@@ -0,0 +1,10 @@
+\page ckpt_learn_ex2_nonintrusive Non-Intrusive Program Example 2
+
+\attention All Non-Intrusive serialize methods <b>MUST</b> be placed in the namespace of type which they serialize.
+
+The full code for this *checkpoint* example can be found here:
```

Comment:
I guess that's `magistrate` now.
```suggestion
The full code for this *magistrate* example can be found here:
```

Reply:
Although, on the other hand, there's still `checkpoint` everywhere in the file names and the code :shrug:

Reply:
Let's stick with `checkpoint` for consistency with the other examples.